src/main/java/net/snowflake/client/core/SFResultSet.java

```diff
@@ -84,17 +67,17 @@
    */
   public SFResultSet(
       SnowflakeResultSetSerializableV1 resultSetSerializable,
+      SFSession session,
```

Comment:
Doesn't statement still have a getSession ?
Why is this SFSession and not SFSessionInterface ?